### PR TITLE
chore: upgrade to Aspect Workflows 5.9.0-rc.9

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: .aspect/workflows/config.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ tasks:
       queue: aspect-small-amd64
   - buildifier:
       queue: aspect-small-arm64
+  # - configure:
```
Comment (gregmagolan) [inline .aspect/workflows/config.yaml:8]: Configure not working on rc.9 on CCI

Comment (jbedard) [reply .aspect/workflows/config.yaml:8]: What was the issue?

Comment (gregmagolan) [reply .aspect/workflows/config.yaml:8]: The Setup step blows up. We don't have coverage for that on silo since we vendor in the CCI yaml.

Comment (gregmagolan) [reply .aspect/workflows/config.yaml:8]: Here's a build with the issue: https://app.circleci.com/pipelines/github/aspect-build/aspect-cli/1287/workflows/e5e4064b-4f92-446d-93bb-9ff4aee78644/jobs/3170

**Review by kormide — APPROVED** [review]

**Pull request merged**